### ISSUE TITLE
I can't install qlauncher on Ubuntu 20.0.3 LTS (server)

### ISSUE THREAD

**Issue by fernoliv** [poseidon-network/qlauncher-linux#42]
My environment met all the hardware requirement and I was running nodes for several months. Now I did new installation from scratch but the installation script is "Killed" and I can't continue installing qlauncher.

Please take a look on the screenshot below:

![Captura de Tela 2021-12-06 às 11 15 05](https://user-images.githubusercontent.com/95630957/144861805-249f58c1-b381-4924-b391-5e3eb01fef7c.png)

**Comment by coderkk** [comment]
I think it install successful.

Maybe you could try check `sudo /opt/qlauncherV3/qlauncher status`.

**Comment by fernoliv** [comment]
> I think it install successful.
> 
> Maybe you could try check `sudo /opt/qlauncherV3/qlauncher status`.

No, it's not. I don't have any containers running, as you can see below:

![Captura de Tela 2021-12-06 às 11 20 40](https://user-images.githubusercontent.com/95630957/144862479-1858b3c1-701d-4dcb-b908-dadf59039119.png)

**Comment by coderkk** [comment]
Can you try restart server or `sudo systemctl restart qlauncher`

**Comment by fernoliv** [comment]
@coderkk thanks for your help, but it didn't take any effect:

![Captura de Tela 2021-12-06 às 11 32 51](https://user-images.githubusercontent.com/95630957/144864367-28d9db12-3371-4a67-b75d-70b4efdf0deb.png)

**Comment by fernoliv** [comment]
I already reinstalled it from scratch several times. Even using docker from the Ubuntu Juju flavors, installing it using "apt install docker.io" or even through the Docker official repositories it is not evolving.

**Comment by coderkk** [comment]
Could you remove the docker.io, because it will conflict with K3S.

**Comment by fernoliv** [comment]
@coderkk I did but the qlauncher depends of Docker, right?

![Captura de Tela 2021-12-06 às 11 57 19](https://user-images.githubusercontent.com/95630957/144868512-b8f5054f-1655-4534-9732-78fd848dafc8.png)

**Comment by coderkk** [comment]
No. K3S has build-in containerd, it no need docker.

**Comment by fernoliv** [comment]
I'm doing now a new installation from scratch without any Docker installation and I'll post here as soon I see how it works.

**Comment by fernoliv** [comment]
I just did a new installation from scratch of Ubuntu 20.04.3 LTS Server without any Docker and I had the same error:

![Captura de Tela 2021-12-06 às 12 18 25](https://user-images.githubusercontent.com/95630957/144872132-2067cfe1-fd5e-4f63-9035-d9609118c275.png)

**Comment by fernoliv** [comment]
Any thoughts or ideas to fix it?

**Comment by fernoliv** [comment]
What other Linux operating system did you guys test the QLauncher? Would be worth test it on CentOS or any other Linux OS for example? I need to have that working as before.

PS: I also tried to download and run it from the binaries of GitHub (https://github.com/poseidon-network/qlauncher-linux/releases/download/0.4.0.2/qlauncher-linux-amd64) and it ended with same behavior (process killed).

I'm running it on top of a Proxmox Virtualization Environment 7.1 with KVM full virtualization (no containers).

**Comment by fernoliv** [comment]
I also installed CentOS Linux release 8.5.2111 and had the same issues, apparently the problem is with systemd on both flavors (Ubuntu and CentOS):

![Captura de Tela 2021-12-06 às 18 15 18](https://user-images.githubusercontent.com/95630957/144924263-230b520e-5090-4cbf-a2de-61ef24c61044.png)
![Captura de Tela 2021-12-06 às 18 15 56](https://user-images.githubusercontent.com/95630957/144924284-5f2f9929-f015-47ab-b305-ec15deac8db1.png)

**Comment by fernoliv** [comment]
@coderkk could you help me to fix that installation issue? What could I do? Test a third or fourth Linux flavor maybe?

**Comment by coderkk** [comment]
Do you have telegram?

**Comment by fernoliv** [comment]
> Do you have telegram?

yeah, @fernoliv
